tools/packaging/packaging.psm1

```diff
@@ -2747,9 +2747,12 @@ function Test-FileWxs
     {
         if ($indexedAssetFiles -inotcontains $file)
         {
+            $name = Split-Path -Path $file -Leaf
+            if ($name -eq '.nupkg.metadata') {
```

Comment:
Is it maybe better to add `'*.nupkg.metadata'` to the blacklist `$dontCopy` when building in build.psm1 [here](https://github.com/PowerShell/PowerShell/blob/master/build.psm1#L2315)

Reply:
I think it might be better in build.psm1 which already has a blacklist